CHANGELOG.md

```diff
@@ -8,6 +8,29 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Tractus-X has switched to YY.0M.MICRO [Calendar Versioning](https://calver.org/) for overarching, quarterly Releases. Semantic Versioning remains unchanged for all components.
 
+## [24.03] - 2024-03-XX
+
+### Added
+### Changed
```

Comment:
EDC (is more or less carry-over, but tested)
SDE
MSO
IRS
Knowledge Agent
Golden Record
Country Risk
MIW
Portal
Policy Hub; Certificate Hub
SD Factory
Digital Twin Registry
Discovery Service (BPN Finder) carry-over but tested
Discovery Finder
Semantic Hub
Trace-X
Digital Product Pass (DPP)

KITs:
Business Partner
Connector
Data Governance
Digital Twin
Knowledge Agent
Traceability
Quality
Eco Pass
Product Carbon Footprint (PCF)
Demand and Capacity Management (DCM)
Online Simulation (OSim)
=> pls confirm QG-approved versions in table style